src/caret_analyze/infra/lttng/records_provider_lttng.py

```diff
@@ -1262,6 +1320,7 @@ def to_records(self):
         )
 
         drop_columns = list(set(pub_sub_records.columns) - set(columns))
+        drop_columns += [s for s in pub_sub_records.columns if s.endswith('callback_start')]
```

Comment:
There do not appear to be any columns added by this line. Please check the implementation.

Reply:
It doesn't seem to be used even in normal implementations.
Fixed in https://github.com/tier4/caret_analyze/pull/516/commits/06d1ef4693a0ca8917bc8f20a9919342f3613aef.